src/components/pages/VotingPage/DropZone/DropZone.tsx

```diff
@@ -19,7 +19,7 @@ const DropZone = ({
       {(provided: DroppableProvided, snapshot: DroppableStateSnapshot) => (
         <div
           className={`drop-zone${snapshot.isDraggingOver ? ' drag-over' : ''}${
-            snapshot.draggingFromThisWith ? ' drag-over' : ''
+            snapshot.draggingFromThisWith ? ' drag-from' : ''
```

Comment:
YES sorry I broke this lmao

Reply:
No worries we got it fixed and that's all that matters